bevy_lint/README.md

```diff
@@ -1,9 +1,168 @@
+<div class = "rustdoc-hidden">
+
 # `bevy_lint`
 
 `bevy_lint` is a custom linter for the [Bevy game engine](https://bevyengine.org), similar to [Clippy](https://doc.rust-lang.org/stable/clippy).
 
+</div>
+
+<div class="rustdoc-alert rustdoc-alert-warning">
+
+> **Warning**
+>
+> This is an unofficial community project, hacked upon by the Bevy CLI working group until it is eventually upstreamed into the main [Bevy Engine organization](https://github.com/bevyengine). Pardon our rough edges!
+
+</div>
+
+## Installation
+
+`bevy_lint` depends on a pinned nightly version of Rust with the `rustc-dev` Rustup component. This is because `bevy_lint` uses [internal `rustc` crates](https://doc.rust-lang.org/nightly/nightly-rustc/) that can only be imported with the permanently-unstable [`rustc_private` feature](https://doc.rust-lang.org/nightly/unstable-book/language-features/rustc-private.html). You can refer to the [compatibility table](#compatibility) to see which version of the linter requires which toolchain.
+
+You can install the toolchain with:
+
+```bash
+rustup toolchain install $TOOLCHAIN_VERSION \
+    --component rustc-dev \
+    --component llvm-tools-preview
+```
+
+For example, you would replace `$TOOLCHAIN_VERSION` with `nightly-2024-10-03` if you were installing `bevy_lint` 0.1.0. Please be aware that you must keep this toolchain installed for `bevy_lint` to function[^keep-toolchain-installed].
```

Comment:
This doesn't make it very clear where to find the correct nightly toolchain. Or can it be any?

Reply:
I added a link to the compatibility table, which specifies which Rust toolchain must be installed to compile and use `bevy_lint`. I also expanded the descriptions of the compatibility table's fields, so it's more clear that it needs to be a _specific_ nightly toolchain.